.gitignore

```diff
@@ -12,7 +12,6 @@
 /dist
 /mkosi.build
 /mkosi.egg-info
-/mkosi.extra
```

Comment:
This seems a bit too restrictive, we should have the freedom to reorder things inside the repository. If users are running from the repository, they'll have to accept that that might involve their images changing when they pull down a new version

Reply:
Agreed, we should have that freedom, but these files are just for testing purposes, no? My thinking was, that this would be better communicated by not having them in the tree proper.

Reply:
With the files moved this isn't necessary anymore, no?